packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -77,7 +86,7 @@ const tab = style({
   alignItems: 'center',
   position: 'relative',
   cursor: 'default',
-  minWidth: 'max'
+  flexShrink: 0
```

Comment:
default of flex layouts is to stretch, so the min/width isn't needed

tabs should not shrink by default, though I imagine a user should be able to override the flex grow so that all tabs take up the same space across the tablist

this will matter more once we get to collapse behavior